packages/encryption/src/ec.ts

```diff
@@ -502,3 +512,37 @@ export function verifyECDSA(
   const contentHash = hashSha256Sync(contentBuffer);
   return verify(signature, contentHash, publicKey);
 }
+
+export function parseRecoverableSignature(signature: string) {
```

Comment:
Seems fine to move to @stacks/ common, we can start a new signature.ts file.
For the long-term we can think about adding another smaller signature specific package which tx+enc can rely on.